zio-http/shared/src/main/scala-2/zio/http/UrlInterpolator.scala

```diff
@@ -68,9 +68,8 @@ private[http] object UrlInterpolatorMacro {
         val exampleParts = staticParts.zipAll(injectedPartExamples, "", "").flatMap { case (a, b) => List(a, b) }
         val example      = exampleParts.mkString
         URL.decode(example) match {
-          case Left(error) =>
-            c.abort(c.enclosingPosition, s"Invalid URL: ${error.getMessage}")
-          case Right(url)  =>
+          case Left(error) => c.abort(c.enclosingPosition, error.getMessage)
+          case Right(_)    =>
```

Comment:
Thank you, the compiler warning from this was annoying 😄

Reply:
It was! 😄